src/Stratis.Bitcoin.Features.SmartContracts.Tests/ExecutorSpecification.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System.Collections.Generic;
+using CSharpFunctionalExtensions;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NBitcoin;
+using Stratis.SmartContracts;
+using Stratis.SmartContracts.Core;
+using Stratis.SmartContracts.Core.State;
+using Stratis.SmartContracts.Core.State.AccountAbstractionLayer;
+using Stratis.SmartContracts.Executor.Reflection;
+using Xunit;
+
+namespace Stratis.Bitcoin.Features.SmartContracts.Tests
+{
+    public class ExecutorSpecification
+    {
+        [Fact]
+        public void Create_Contract_Success()
```

Comment:
Success is implied ?  Can this title be improved?

Reply:
It is implied. The idea is to test the path of successful contract execution.